terraform/modules/github_repository/main.tf

```diff
@@ -44,10 +65,6 @@ resource "github_repository_collaborators" "admins" {
     permission = "admin"
     team_id    = "service-accounts" # id: 6206668
   }
-  user {
-    permission = "admin"
-    username   = "cloudeteerbot"
```

Comment:
not sure if this is working or not.

if terraform first removes cloudeteerbot before service-accounts is added, the job is screwed.